setup.py

```diff
@@ -26,7 +26,6 @@
         'boto3==1.14.7',
         'munch==2.5.0',
         'pytest==5.3.5',
-        'pytest-reportportal==1.10.0',
```

Comment:
We could keep `pytest-reportportal` there (without selecting particular version), but removal works as well (we use requirements.txt file as a primary way to perform installation).

Reply:
I don't think we can leave `pytest-reportportal` without a version because the latest version is incompatible with our instance of report portal.

Reply:
> I don't think we can leave `pytest-reportportal` without a version because the latest version is incompatible with our instance of report portal.

As noted in our docs (and assumed by many decisions across ocs-ci codebase), we install ocs-ci via `requirements.txt` files. So by referencing the report portal fork in `requirements.txt`  file, we don't need to specify version in `setup.py` file, because when this information is processed, report portal is already installed and pip won't try to install another version because the requirement is already satisfied.